README.org

```diff
@@ -6,97 +6,141 @@ A sensible starting point for hacking your own Emacs configuration.
+The goal of this configuration is to make it easier to write your own
+Emacs configuration while using pre-made configuration parts
+maintained by the community. Instead of using a monolithic,
+all-encompassing approach, we strive to ensure that all parts of this
+configuration are optional or interchangeable.
 
-You should even be able to use the configuration modules we provide with your own =init.el= file without using this base configuration repo!
+You should even be able to use the configuration modules we provide
+with your own =init.el= file without using this base configuration
+repo!
 
 * Modules
 
-Here is a list of the built-in modules that you may load.  Follow the links to each to get more information about how they can be configured!
-
-- [[modules/rational-defaults.el][rational-defaults]]: Sensible default settings for Emacs
-- [[modules/rational-ui.el][rational-ui]]: Extra UI configuration for a better experience (mode line, etc)
-- [[modules/rational-completion.el][rational-completion]]: A better selection framework configuration based on Vertico
-- [[modules/rational-evil.el][rational-evil]]: An evil-mode configuration
-- [[modules/rational-windows.el][rational-windows]]: Window management configuration
-- [[modules/rational-use-package.el][rational-use-package]]: Configuration for use-package if you prefer it over straight.el
+Here is a list of the built-in modules that you may load. They are
+located in directory =$RATIONAL_EMACS_HOME/modules=, which are in the
+directory your =git= clone from listing [[li#git_clone]]. Follow the links
+to each to get more information about how they can be configured!
+
+- [[file:modules/rational-defaults.el][rational-defaults]] :: Sensible default settings for Emacs
+- [[file:modules/rational-ui.el][rational-ui]] :: Extra UI configuration for a better experience (mode
+  line, etc)
+- [[file:modules/rational-completion.el][rational-completion]] :: A better selection framework configuration
+  based on =Vertico=
+- [[file:modules/rational-evil.el][rational-evil]] :: An =evil-mode= configuration
+- [[file:modules/rational-windows.el][rational-windows]] :: Window management configuration
+- [[file:modules/rational-use-package.el][rational-use-package]] :: Configuration for =use-package= if you
+  prefer it over =straight.el=
 
 Modules that we will be adding in the future:
 
-- rational-desktop: A desktop environment centered around EXWM
-- rational-present: Tools for giving presentations
-- rational-screencast: Tools for doing screencasts
-- rational-workspace: An improved workspace experience based on =tab-bar-mode=
-- rational-shell: A starter configuration for =eshell= and =vterm=
+- rational-desktop :: A desktop environment centered around =EXWM=
```

Comment:
These definition lines look great in the rendered output on GitHub, I didn't know about that!